packages/fluentui/docs/src/prototypes/Roster/participantData.tsx

```diff
@@ -0,0 +1,578 @@
+import * as React from 'react';
+import Participant from './participant';
+import { AcceptIcon } from '@fluentui/react-icons-northstar';
+
+const participants = [
+  {
+    id: 1,
+    title: (
+      <Participant
+        name="Irving Kuhic"
+        role="Software Engineer"
+        isMuted
+        image="https://s3.amazonaws.com/uifaces/faces/twitter/ariffsetiawan/128.jpg"
```

Comment:
For the perf test to be relevant there should be no additional network requests.
Use base64 encoded images, similar to this:
https://github.com/microsoft/fluentui/blob/8247d915e42b98aceeeaaf0061c1ce3773bd17fb/packages/fluentui/docs/src/examples/components/List/Performance/ListCommon.perf.tsx#L4-L34